chart/templates/etcd-statefulset.yaml

```diff
@@ -91,6 +91,9 @@ spec:
         - --data-dir=/var/etcd/data/new.etcd
         - --storage-provider={{ .Values.backup.storageProvider }}
         - --store-prefix=etcd-{{ .Values.role }}
+{{- if .Values.backup.embeddedEtcdQuotaBytes }}
+        - --embeddedEtcdQuotaBytes={{ .Values.backup.embeddedEtcdQuotaBytes }}
```

Comment:
Shouldn't this be `embedded-etcd-quota-bytes`?

Reply:
Nice catch. Thank you.

Reply:
I'll do the correction with helm chart upgrade tomorrow.